pkg/virt-launcher/virtwrap/network/dhcpv6/dhcpv6.go

```diff
@@ -29,12 +26,7 @@ type DHCPv6Handler struct {
 
 func SingleClientDHCPv6Server(
 	clientIP net.IP,
-	serverIfaceName string,
-	dnsIPs [][]byte,
-	routes *[]netlink.Route,
-	searchDomains []string,
-	mtu uint16,
```

Comment:
This may be problematic.
Is the DHCPv4 server aware if IPv6 is enabled? There is a different minimum value for the [IPv6 mtu](https://tools.ietf.org/html/rfc2460#section-5).

Reply:
Really good point.

Reply:
The virt-launcher/virt-handler is taking the MTU as is from the pod interface. The MTU or any other information taken from the pod interface is not validated, as we trust the cni.
So IIUC, it is enough for dhcpv4 to just pass the mtu as is. There is no need for ipv6 awareness.
I will update the commit message to clarify it.